qiskit_sphinx_theme/layout.html

```diff
@@ -1,263 +1,142 @@
-              <a class="resource-option with-down-orange-arrow">
-                Applications
-              </a>
-              <div class="resources-dropdown-menu">
-                <a class="nav-dropdown-item" href="{{ theme_variables.external_urls['ml'] }}">
-                  <span class="dropdown-title">Machine learning</span>
-                  <p>QSVM, VQC (Variational Quantum Classifier), and QGAN (Quantum Generative
-                    Adversarial Network) algorithms.</p>
-                </a>
-                <a class="nav-dropdown-item" href="{{ theme_variables.external_urls['nature'] }}">
-                  <span class="dropdown-title">Nature</span>
-                  <p>Quantum applications in chemistry, physics, and biology.</p>
-                </a>
-                <a class="nav-dropdown-item" href="{{ theme_variables.external_urls['finance'] }}">
-                  <span class="dropdown-title">Finance</span>
-                  <p>Uncertainty components for stock/securities problems, Ising translators for
-                    portfolio optimizations and data providers to source real or random data.</p>
-                </a>
-                <a class="nav-dropdown-item" href="{{ theme_variables.external_urls['optim'] }}">
-                  <span class="dropdown-title">Optimization</span>
-                  <p>High-level optimization problems that are ready to
-                    run on simulators and real quantum devices</p>
-                </a>
-              </div>
-          </li>
-          <li>
-              <a href="{{ theme_variables.external_urls['experiments'] }}" target="_blank">Experiments</a>
-          </li>
-          <li>
-            <div id="resourcesDropdownButton" data-toggle="resources-dropdown" class="resources-dropdown">
-              <a class="resource-option with-down-orange-arrow">
-                Resources
-              </a>
-              <div class="resources-dropdown-menu">
-                <a class="doc-dropdown-option nav-dropdown-item" href="{{ theme_variables.external_urls['slack'] }}">
-                  <span class="dropdown-title">Slack support</span>
-                  <p></p>
-                </a>
-                <a class="doc-dropdown-option nav-dropdown-item" href="{{ theme_variables.external_urls['textbook'] }}">
-                  <span class="dropdown-title">Qiskit Textbook</span>
-                  <p></p>
-                </a>
-                <a class="doc-dropdown-option nav-dropdown-item" href="{{ theme_variables.external_urls['events'] }}">
-                  <span class="dropdown-title">Qiskit events</span>
-                  <p></p>
-                </a>
-            </div>
-          </li>
-          <li>
-            <a href="{{ theme_variables.external_urls['github'] }}" target="_blank">GitHub</a>
-          </li>
-        </ul>
-      </div>
-
-      <a class="main-menu-open-button" href="#" data-behavior="open-mobile-menu"></a>
-    </div>
-
-  </div>
-</div>
-
-
```

Comment:
Why were these lines removed?

Reply:
This is the old top menu that is now replaced by the unified top menu